### ISSUE TITLE
Compass is installed, but Yeoman doesn't think so. 

### ISSUE THREAD

**Issue by dalanmiller** [yeoman/yeoman#632]
For some reason Yeoman upon installation is saying that Compass is installed and typing compass from the CLI works just fine. It seems like similar issues to this have popped up in the past but were resolved. 

I've tried both `sudo gem install compass` and `gem install compass` but neither seems to make a difference.

Anyone have any idea how I can fix this? 

```
bash-3.2$ which compass
/Users/me/.rbenv/shims/compass
```

Yeoman tells me that Compass is installed here and it's available on $PATH

```
Before I can assist you I will need to confirm you have the necessary requirements. 

Below you will find the results of my short system audit:

✓ Command Line Tools for Xcode check.
✓ Homebrew check.
✓ git smashing!
✓ NodeJS check.
✓ ruby check.
✓ RubyGems check.
✓ Compass check.
✓ PhantomJS check.
✓ jpegtran check.
✓ optipng check.
✓ yeoman global npm module extraordinary!
```

TL;DR trying `yeoman server` just results in failing and telling me Compass is not installed.

```
bash-3.2$ yeoman server
Running "server" task

Starting static web server on port 3501
  - /Users/me/...
I'll also watch your files for changes, recompile if neccessary and live reload the page.
Hit Ctrl+C to quit.

Running "clean" task

Running "coffee:compile" (coffee) task
File ... created.
...

Running "compass:dist" (compass) task

...

  /Users/me/.rbenv/versions/1.9.2-p290/lib/ruby/gems/1.9.1/gems/compass-0.13.alpha.0/frameworks/blueprint/stylesheets
  /Users/me/.rbenv/versions/1.9.2-p290/lib/ruby/gems/1.9.1/gems/compass-0.13.alpha.0/frameworks/compass/stylesheets
  Compass::SpriteImporter)
   create temp/styles/transit.css 
<FATAL> You need to have Compass installed. </FATAL>
```

My path

```
bash-3.2$ echo $PATH $NODE_PATH
/Users/me/.rbenv/shims:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/bin:/usr/X11/bin:/opt/local/bin:/usr/local/git/bin
```
